src/store/actions.ts

```diff
@@ -229,17 +257,39 @@ export const cancelOrder = (order: SignedOrder) => {
     };
 };
 
-export const submitLimitOrder = (amount: BigNumber, price: BigNumber, side: OrderSide) => {
-    return async (dispatch: any, getState: any) => {
+export const resetSteps = () => {
```

Comment:
Pretty sure that this could be a "normal" (non-thunk) action:

```typescript
export const resetSteps = createAction('RESET_STEPS');
```

that then you just handle in the reducer by setting pending/done steps to `[]` and current step to `null`.

Reply:
:+1:  will do that and return the initial state in the reducer.